uber/templates/jobs/index.html

```diff
@@ -1,9 +1,27 @@
 {% extends "base-admin.html" %}
 {% block title %}Shifts{% endblock %}
+{% block page_styles %}
+<!--inside page_style -->
+<link rel="stylesheet" href="../static/styles/fullcalendar.min.css" />
+<link rel="stylesheet" href="../static/styles/fullcalendar.print.css" media="print"/>
```

Comment:
We really need to document how to plug thing into our Javascript and CSS minification and consolidation.  Until then adding these kinds of dependencies is fine.